### PR TITLE
Created new ubuntu base image with curl and ca_certificates

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

File: reproducible/ubuntu/BUILD [hunk 1]
```diff
@@ -12,16 +12,20 @@ NODEJS_ENV = {
...
 docker_build(
-    name = "ubuntu",
+    name = "ubuntu_vanilla",
     cmd = "/bin/bash",
     env = UBUNTU_ENV,
     tars = ["@ubuntu_tar_download//file"],
 )
 
 docker_build(
+    name = "ubuntu",
```
Comment (dlorenc) [inline reproducible/ubuntu/BUILD]: Should the name of this be "ubuntu_build" still?

Comment (aaron-prindle) [reply reproducible/ubuntu/BUILD]: build referenced build-essential.  I am happy to change any of the names though, currently it is:
ubuntu_vanilla -> the image from the dl'd tar
ubuntu -> ubuntu_vanilla + curl, ca-certificates, and netbase
ubuntu_build -> ubuntu + build-essential and pythhon2.7

Comment (dlorenc) [reply reproducible/ubuntu/BUILD]: Yeah maybe part of the confusion is that there are dockerfile_build tars, that then get duplicated into docker_build rules (so the same thing is getting represented by two names) in a few cases. Maybe wrapping up some of that into a macro would help?

Comment (aaron-prindle) [reply reproducible/ubuntu/BUILD]: Should docker_build and dockerfile_build both be necessary for these steps and a macro required (It is currently I believe due to the output format)?  I would think that ideally I could use dockerfile_build and docker_build interchangeably if I wanted to use a dockerfile in my build.

This PR is doing both for each image currently as docker_build needs to be done on the intermediate image to run structure_test on it.

Comment (dlorenc) [reply reproducible/ubuntu/BUILD]: Maybe just leave a comment at the top of the build file explaining what you explained in the reply up there. ubuntu_vanilla is loaded from a .tar.gz, ubuntu is that plus some packages, ubuntu_build is that plus more packages...

**Pull request merged**